Team8923_PowerPlay/src/main/java/org/firstinspires/ftc/team8923_PowerPlay/Constants.java

```diff
@@ -24,8 +24,9 @@ public class Constants {
     public static final double ANGLE_TOLERANCE_DEG = 5.0;
     public static final double POSITION_TOLERANCE_MM = 2*25.4;
 
+    // IMU Constants
+    public static final double AGGRESSION = 0.05;
```

Comment:
What does this mean?

Reply:
It was created for imu strafing but is now deleted